noq_django/noq_django/api/host_api.py

```diff
@@ -87,6 +87,31 @@ def appoint_pending_booking(request, booking_id: int):
     except BookingStatus.DoesNotExist:
         raise HttpError(404, detail="Booking status does not exist.")
 
+@router.patch("/pending/{booking_id}/decline", response=BookingSchema, tags=["host-manage-requests"])
+def decline_pending_booking(request, booking_id: int):
+    host = Host.objects.get(users=request.user)
+    booking = get_object_or_404(Booking, id=booking_id, product__host=host, status__description='pending')
+
+    try:
+        booking.status = BookingStatus.objects.get(description='rejected')
```

Comment:
Statusarna som finns i dagsläget är

```
    statuses = [
        {"id": 1, "description": "pending"},
        {"id": 2, "description": "declined"},
        {"id": 3, "description": "accepted"},
        {"id": 4, "description": "checked_in"},
    ]
```

så rejected hade behövt ändras till declined. alternativt byta namn på declined till rejected.
i dagsläget genereras detta av generate data scriptet eventuellt ska det dock skapas/kollas existens vid Init.

Reply:
Thank you for the review. I'll change the status to declined instead of rejected.